src/app/(main)/layout.tsx

```diff
@@ -4,6 +4,7 @@ import '../globals.css';
 import { auth } from '@/config/firebase';
 import { useRouter } from "next/navigation";
 import { useState, useEffect } from 'react';
+import { getFunctions, httpsCallable } from 'firebase/functions';
```

Comment:
Minor comment but there's a function callApi() in src/config/firebase for this, simplifying it